src/appkit.cs

```diff
@@ -3390,6 +3444,9 @@ interface NSCollectionViewGridLayout
 	}
 
 	[Mac (10,11)]
+#if XAMCORE_4_0
```

Comment:
that pattern should not be used, otherwise people will continue using the incorrect default ctor
instead create/reuse a `NSCompat.cs` file and add a *obsolete* default `.ctor` under `!XAMCORE_4_0`